steps/generate_dataset_from_htme.py

```diff
@@ -335,6 +362,51 @@ def get_spark_session():
     return spark
 
 
+def get_todays_date():
+    return datetime.now().strftime('%Y-%m-%d')
+
+
+def log_start_of_batch(correlation_id, dynamodb=None):
+    """Logging start of batch in metadata audit table as In Progress"""
+    if not dynamodb:
+        dynamodb = get_resource('dynamodb')
+    data_pipeline_audit_table = "${data_pipeline_audit_table}"
+    table = dynamodb.Table(data_pipeline_audit_table)
+    run_id = 1
+    response = table.query(
```

Comment:
Can we deal with exceptions here as there can be connectivity issues sometimes while querying/inserting/updating records in dynamo db